.github/workflows/register-buildpack.yml

```diff
@@ -0,0 +1,260 @@
+name: Register Buildpack
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  register:
+    env:
+      NAMESPACE_REPO: registry-namespaces
+      GITHUB_OWNER: buildpacks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node Dependencies
+        working-directory: .github/workflows/scripts/validate
+        shell: bash
+        run: |
+          npm install
+      - id: detect_validate
+        name: Validate Buildpack Issue
+        uses: actions/github-script@v2
+        with:
+          result-encoding: string
+          script: |
+            const path = require('path')
+            const scripts = require(path.resolve('.github/workflows/scripts/validate/index.js'))
+            let result
+            try {
+              result = scripts.validateIssue({context})
+            } catch (error) {
+              console.error(error)
+              process.exit(1)
+            }
+            console.log(`::set-env name=BUILDPACK::${JSON.stringify(result)}`)
+
+      - uses: actions/github-script@v2
+        name: Retrieve Owners
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}
+          script: |
```

Comment:
Can we make all scripts external to a JS file like We do for validate? It makes it easier to review, keeps a consistent pattern, and can more independently tested?

Reply:
Sounds good.  I'm new to GitHub actions, and I was struggling to understand if there's a preferable way to write node scripts inline vs external files.  It also isn't clear to me if there's a way to test inline scripts.   Anyway, seems reasonable to me.